docs/octopus-rest-api/octopus-cli/list-machines.md

```diff
@@ -5,7 +5,9 @@ position: 200
 ---
 
```

Comment:
page title is bad, missing intro text

Reply:
For other sections, I've converted the titles like so `list-machines` to `List Machines`, I'll  do the same in this section, but let me know if you meant something else.

Reply:
> For other sections, I've converted the titles like so `list-machines` to `List Machines`, I'll do the same in this section, but let me know if you meant something else.

Awesome - yep, that's what I meant. 
(In hindsight, I should've put more words - these were just notes to self rather than full blown comments)